src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -67,15 +69,21 @@ public static SusiThought  scrapeQuora(String profile) {
 		JSONObject quoraProfile = new JSONObject();
 
 		Document userHTML = null;
-		String url = "https://www.quora.com/profile/" + profile;
+		String QUORA_URL_BASE = "https://www.quora.com";
+		String url = QUORA_URL_BASE + "/profile/" + profile;
 
 		try {
-			userHTML = Jsoup.connect(url).get();
+			userHTML = Jsoup.connect(url)
+			                .header("Accept-Encoding", "gzip, deflate")
+			                .userAgent("Mozilla")
+			                .maxBodySize(0)
+			                .timeout(600000)
+			                .get();
 		} catch (IOException e) {
 			e.printStackTrace();
 		}
 
-		String bio = userHTML.getElementsByClass("qtext_para").text();
+		String bio = userHTML.getElementsByClass("ProfileDescription").text();
```

Comment:
Will throw `NullPointerException` if previous try block throws `IOException`. Please take appropriate steps in the previous catch block.